components/edit-collective/Menu.js

```diff
@@ -69,9 +70,9 @@ const SECTION_LABELS = defineMessages({
     id: 'editCollective.menu.export',
     defaultMessage: 'Export',
   },
-  [EDIT_COLLECTIVE_SECTIONS.EXPENSES]: {
-    id: 'editCollective.menu.expenses',
-    defaultMessage: 'Expenses Policy',
+  [EDIT_COLLECTIVE_SECTIONS.POLICIES]: {
+    id: 'editCollective.menu.policies',
+    defaultMessage: 'Collective Policies',
```

Comment:
Given that this section is also displayed on funds, and might one day be displayed on events, maybe we can simply call it `Policies`?